README.md

```diff
@@ -195,6 +195,57 @@ buildctl build \
 
 `--local` exposes local source files from client to the builder. `context` and `dockerfile` are the names Dockerfile frontend looks for build context and Dockerfile location.
 
+##### Hint: `dockerfile.v0` frontend with `buildctl build --opt` option
```

Comment:
I think you also need to update the table of contents.

Reply:
Thanks for the reminder.